src/home_robot/home_robot/mapping/semantic/instance_tracking_modules.py

```diff
@@ -53,6 +67,56 @@ def __init__(
         self.mask = mask
         self.point_cloud = point_cloud
         self.category_id = category_id
+        self.bounds = bounds
+
+    @staticmethod
+    def create_from_observations(
```

Comment:
`InstanceMemory.process_instances_for_env` performs the same thing.

Reply:
Ah maybe i was misinterpreting the input parameters here?

Reply:
Resolving this because i think we have it working now

Reply:
Can this be removed now?

Reply:
Seems like this is unused though. May remove this now/later?